src/state/index.ts

```diff
@@ -18,13 +29,27 @@ import swap from './swap/reducer'
 import transactions from './transactions/reducer'
 import user from './user/reducer'
 
+const migrations = {
+  0: (state) => {
+    return {
+      ...state,
+      user: {
+        ...state.user,
+        theme: state.user.isDark ? Theme.DARK : Theme.LIGHT,
```

Comment:
Don't set to light, instead let default to be `undefined`, or you can change it back to `null`.
I think it's safe to take out isDark and no need to save it in next version 
`const { isDark, ...user } = state.user`

Reply:
Done